cmake/modules/SwiftAndroidSupport.cmake

```diff
@@ -1,57 +1,14 @@
-function(swift_android_prebuilt_host_name prebuilt_var_name)
-  # Get the prebuilt suffix to create the correct toolchain path when using the NDK
-  if(CMAKE_HOST_SYSTEM_NAME STREQUAL Darwin)
-    set(${prebuilt_var_name} darwin-x86_64 PARENT_SCOPE)
-  elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL Linux)
-    set(${prebuilt_var_name} linux-x86_64 PARENT_SCOPE)
-  elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
-    set(${prebuilt_var_name} Windows-x86_64 PARENT_SCOPE)
-  else()
-    message(SEND_ERROR "cannot cross-compile to android from ${CMAKE_HOST_SYSTEM_NAME}")
-  endif()
-endfunction()
-
-function(swift_android_libgcc_for_arch_cross_compile arch var)
-  set(paths)
-  if(NOT "${SWIFT_ANDROID_NDK_PATH}" STREQUAL "")
-    list(APPEND paths "${SWIFT_SDK_ANDROID_ARCH_${arch}_PATH}/../lib/gcc/${SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_TRIPLE}/${SWIFT_ANDROID_NDK_GCC_VERSION}.x")
-  endif()
-  set(${var} ${paths} PARENT_SCOPE)
-endfunction()
-
 function(swift_android_sysroot sysroot_var_name)
   if(NOT "${SWIFT_ANDROID_NDK_PATH}" STREQUAL "")
-    swift_android_prebuilt_host_name(prebuilt_build)
-    set(${sysroot_var_name} "${SWIFT_ANDROID_NDK_PATH}/toolchains/llvm/prebuilt/${prebuilt_build}/sysroot" PARENT_SCOPE)
+    string(TOLOWER ${CMAKE_HOST_SYSTEM_NAME} platform)
+    set(${sysroot_var_name} "${SWIFT_ANDROID_NDK_PATH}/toolchains/llvm/prebuilt/${platform}-x86_64/sysroot" PARENT_SCOPE)
```

Comment:
This might be change shortly when Google releases native NDK for Darwin-arm64. I think the previous function was better in case further changes were needed. The `Windows`/`windows` change is fine, but it was probably was working because Windows filesystem is normally case insensitive.

Reply:
We can always change this later if we ever support that mooted arm64 NDK.